packages/dds/tree/src/simple-tree/schemaCreationUtilities.ts

```diff
@@ -61,32 +66,49 @@ export function singletonSchema<TScope extends string, TName extends string | nu
  * Converts an enum into a collection of schema which can be used in a union.
  * @remarks
  * Currently only supports `string` enums.
+ * The string value of the enum is used as the name of the schema: ensure that its stable and unique.
```

Comment:
Nit:
```suggestion
 * The string value of the enum is used as the name of the schema: you must ensure that its stable and unique.
```

Reply:
I don't think there is a clear "you" here (ex: neither of us as reviewers or implementers are the "you"). Perhaps "callers" would make more sense?